sdk/core/core-tracing/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.0-preview.12 (Unreleased)
+## 1.0.0 (Unreleased)
+
+- This release marks the general availability of the `@azure/core-tracing` package.
```

Comment:
put a note about OpenTelemetry also becoming GA?

Reply:
Done (and same for the other comment)